examples/hello_world/build/hello_world.py

```diff
@@ -0,0 +1,35 @@
+import os
+import typer
+from datetime import datetime
+
+
+app = typer.Typer()
```

Comment:
I am so sorry but it looks like `typer` is still too early :(
With [click](https://click.palletsprojects.com/en/7.x/), [as shown here in `mlcommons_box`](https://github.com/mlperf/mlbox/blob/master/mlcommons_box/mlcommons_box/main.py#L13-L42) we can add tests with the in-built test framework that click provides.
[Testing reference for `mlcommons_box`](https://github.com/mlperf/mlbox/blob/master/mlcommons_box/mlcommons_box/tests/test_mlcommons_box_cli.py)